src/System/Wlog/PureLogging.hs

```diff
@@ -115,17 +116,50 @@ runNamedPureLog
 runNamedPureLog (NamedPureLogger action) =
     askLoggerName >>= (`usingLoggerName` runPureLog action)
 
--- | Similar to 'launchPureLog', but provides logger name from current context.
+{- | Similar to 'launchPureLog', but provides logger name from current context.
+
+Running the 'NamedPureLogger' gives us the pair  of target and the list of 'LogEvent's,
+wrapped in 'Monad' from where using the fact that @(,)@ is 'Functor' logging can be triggered.
+
+==== __Example__
+
+@
+  newtype PureSmth a = ...
+      deriving (MonadSmth, ...)
```

Comment:
I think example also should contain string `Instance MonadSmth m => MonadSmt (NamedLoggerName m)` or something like this. I guess without such instance it won't work...